frontend/src/components/phones/onboarding/PurchasePhonesPlan.tsx

```diff
@@ -124,8 +132,16 @@ const PricingToggle = (props: PricingToggleProps) => {
           tabIndex={0}
           className={styles["pick-button"]}
         >
-          {l10n.getString("phone-onboarding-step1-button-cta")}
+          {l10n.getString("phone-onboarding-step1-button-cta-2")}
         </LinkButton>
+        <a
+          className={styles["learn-more"]}
+          href={
+            "https://support.mozilla.org/en-US/kb/how-do-i-upgrade-my-subscription"
```

Comment:
suggestion (blocking): When linking to SUMO, don't hard-code the `en-US` locale into the url. SUMO will auto-detect the right locale based on the browser's `Accept-Language` header.

```suggestion
            "https://support.mozilla.org/kb/how-do-i-upgrade-my-subscription"
```